playbooks/ufw-playbook.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- hosts: frontend_servers
+  roles:
+    - franklinkim.ufw
```

Comment:
Just noticed ansible had a built-in ufw module. Does the role offer more convenience?
http://docs.ansible.com/ansible/ufw_module.html